src/widgets/_menus.scss

```diff
@@ -25,6 +25,10 @@
         background-color: #{'alpha(@fg_color, 0.15)'};
     }
 
+    &:selected {
+        @extend %selected_item;
+    }
```

Comment:
Can you give an example of where menuitems or modelbuttons are selected?

Reply:
Some third-party apps have this distinction for coloring. Should I remove this then?

Reply:
Going to remove since it's not used in elementary.